internal/handlers/jsonb1/msg_count.go

```diff
@@ -33,29 +32,15 @@ func (s *storage) MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
 
 	unimplementedFields := []string{
 		"skip",
-		"returnKey",
```

Comment:
started to support this or it was added by a mistake?

Reply:
It was a part of "find or count" that was copied to count by mistake